tests/Doctrine/Tests/Inflector/Rules/English/EnglishFunctionalTest.php

```diff
@@ -299,60 +248,44 @@ public function dataSampleWords(): array
             ['pliers', 'pliers'],
             ['pokemon', 'pokemon'],
             ['police', 'police'],
-            ['police', 'police'],
-            ['police', 'police'],
-            ['police', 'police'],
             ['polish', 'polish'],
             ['portfolio', 'portfolios'],
             ['portuguese', 'portuguese'],
             ['potato', 'potatoes'],
-            ['potato', 'potatoes'],
             ['powerhouse', 'powerhouses'],
             ['prize', 'prizes'],
             ['proceedings', 'proceedings'],
             ['process', 'processes'],
             ['progress', 'progress'],
             ['query', 'queries'],
             ['quiz', 'quizzes'],
-            ['quiz', 'quizzes'],
             ['rabies', 'rabies'],
             ['radius', 'radii'],
             ['rain', 'rain'],
             ['reflex', 'reflexes'],
             ['research', 'research'],
             ['rhinoceros', 'rhinoceros'],
             ['rice', 'rice'],
-            ['rice', 'rice'],
-            ['rice', 'rice'],
             ['roof', 'roofs'],
             ['runner-up', 'runners-up'],
             ['safe', 'safes'],
             ['salesperson', 'salespeople'],
             ['salmon', 'salmon'],
-            ['salmon', 'salmon'],
             ['sand', 'sand'],
             ['sarawakese', 'sarawakese'],
             ['save', 'saves'],
             ['scarf', 'scarves'],
             ['scissors', 'scissors'],
-            ['scissors', 'scissors'],
             ['scratch', 'scratches'],
-            ['sea bass', 'sea bass'],
```

Comment:
We should keep this one.

Reply:
Oh yes. Sorry about that. Late night code!

Reply:
Done!